Sources/Extensions/UIKit/UIViewExtensions.swift

```diff
@@ -10,30 +10,6 @@ public extension UIView {
         translatesAutoresizingMaskIntoConstraints = !autoLayout
     }
 
-    var compatibleTopAnchor: NSLayoutYAxisAnchor {
-        if #available(iOS 11.0, *) {
-            return safeAreaLayoutGuide.topAnchor
-        } else {
-            return topAnchor
-        }
-    }
-
-    var compatibleBottomAnchor: NSLayoutYAxisAnchor {
-        if #available(iOS 11.0, *) {
-            return safeAreaLayoutGuide.bottomAnchor
-        } else {
-            return bottomAnchor
-        }
-    }
-
-    var safeLayoutGuide: UILayoutGuide {
```

Comment:
It's probably a breaking change, we use it in the app and in Charcoal I think.

Reply:
I see, i opted to remove it as I could have kept it and just return the `safeLayoutAreaGuide`, but that (together with the other stuff) mean that we use custom properties instead of the vanilla iOS properties.

I guess we need to update charcoal and any other dependency to FinniversKit to drop iOS 10 as well.

We can potentially restore this if it's a major issue for fixing the app release

Reply:
But we are not including this in the current release right? So the current release is not broken

Reply:
yeah, this is not part of the current release

Reply:
Yes, we just need to update main app and Charcoal so they will not break next time someone wants to update FinniverKit

Reply:
the main app is on iOS 11+, Now to compile it thought we need Charcoal, Finnbox to be updated to iOS 11+.

Then I guess we should release FinniversKit, the unreleased changes right now are:

```
## Changes since 3.2.0 to master (squash)

- #538: Extend empty view with image support
- #540: Handle page control value changes
- #539: Remove conditional code `if #available(iOS 11.0)`
- #536: Integration with `pr_changelog`
- #531: Drop iOS 10
```

Would that mean we go for `4.0`?

Reply:
Yes 👍 

And then update main app and Charcoal to use 4.0